src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml

```diff
@@ -87,7 +87,7 @@ limitations under the License.
     <attribute side="server" code="0x01" define="REBOOT_COUNT" type="INT16U" min="0x0000" max="0xFFFF" writable="false" default="0x0000" optional="false">RebootCount</attribute>
     <attribute side="server" code="0x02" define="UPTIME" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="true">UpTime</attribute>
     <attribute side="server" code="0x03" define="TOTAL_OPERATIONAL_HOURS" type="INT32U" min="0x00000000" max="0xFFFFFFFF" writable="false" default="0x00000000" optional="true">TotalOperationalHours</attribute>
-    <attribute side="server" code="0x04" define="BOOT_REASONS" type="ENUM8" writable="false" optional="true">BootReasons</attribute>
+    <attribute side="server" code="0x04" define="BOOT_REASONS_TYPE" type="BootReasonType" writable="false" optional="true">BootReasons</attribute>
```

Comment:
Can we please make the XML changes in a separate PR from the range enforcement, so those can bre reviewed independently?  They're not related.

Reply:
removed them from this PR